src/server/command/worldedit/blueprint.zig

```diff
@@ -0,0 +1,193 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+const vec = main.vec;
+const Vec3i = vec.Vec3i;
+
+const List = main.List;
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+
+pub const description = "Operate on blueprints.";
+pub const usage = "/blueprint <save FILENAME|delete FILENAME|load FILENAME|list>";
```

Comment:
Wouldn't it be more readable as separate lines? (Also quick reminder to avoid capslock)
```
/blueprint save <file name>
/blueprint delete <file name>
/blueprint load <file name>
/blueprint list
```
This is how other commands deal with subcommands in the help menu.

Reply:
Caps lock here is for different reason than in variable names, it was introduced as part of unix command line argument naming convention where positional arguments are noted in caps lock.

```
$ ls --help
Usage: ls [OPTION]... [FILE]...
List information about the FILEs (the current directory by default).
Sort entries alphabetically if none of -cftuvSUX nor --sort is specified.
``` 

I'll change it as requested.